src/op_analytics/transforms/interop/README.md

```diff
@@ -0,0 +1,72 @@
+interop
+=======
+
+Objective
+---------
+
+Find how many tokens of different bridgeable types are created each day. 
+
+
+Methodology
+-----------
+
+We consider 3 token types:
+
+- SuperchainERC20
+- NTT 
+- OFT
+
+### Token Contract Addresses
+
+For each of these we first focus on transfers. We look at logs to find when the token is
+transferred/sent/deliverd and to find the contract address executing the transfer.
+
+Using the historical events (fact tables) we maintain a first seen dimension table. This table
+contains the contract address for each token and the first time a related event was emitted for it.
+
+The purpose of each dimension table is to hold a list of contract addresses that we are sure
+belong to each of the three different token types.
+
+### Contract Creation Traces
+
+We use traces to find out when a token contract was created. 
+
+For NTT and OFT we inner join with the token contract address dimension table to keep only the
+create traces for those token contracts.
+
+For SuperchainERC20 contracts it is more nuanced. The only events we have are transfer events, which
```

Comment:
Same for here as well. I think once we are sure of it, in the future we can even create more specific subset of Superchainerc20